tests/Unit/Infrastructure/Auth/CsrfCheckTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace OpenCFP\Test\Unit\Infrastructure\Auth;
+
+use Mockery;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use OpenCFP\Infrastructure\Auth\CsrfValidator;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Csrf\CsrfTokenManager;
+
+/**
+ * @covers \OpenCFP\Infrastructure\Auth\CsrfValidator
+ */
+class CsrfValidatorTest extends \PHPUnit\Framework\TestCase
+{
+    use MockeryPHPUnitIntegration;
+
+    public function testIsFinal()
+    {
+        $reflection = new \ReflectionClass(CsrfValidator::class);
+        $this->assertTrue($reflection->isFinal());
+    }
+
+    public function testReturnsTrueWhenTokenMangerReturnsTrue()
+    {
+        $manager = Mockery::mock(CsrfTokenManager::class);
+        $manager->shouldReceive('isTokenValid')->andReturn(true);
```

Comment:
Can we also specify some expectations towards what is passed to `CsrfTokenManager::isValid()`?

More specifically, can we assert that the parameters

* `token`
* `token_id`

are fetched from the `Request` object, and that we pass an instance of `CsrfToken` to `isTokenValid()` which composes these values?